packages/terra-functional-testing/src/reporters/spec-reporter/merge-results.js

```diff
@@ -34,7 +34,7 @@ const getResultsFileName = (options) => {
     theme,
   } = options;
 
-  const keys = [locale, theme, formFactor, browser, packageName].filter(key => !!key);
+  const keys = [locale, theme, formFactor, browser, packageName.split('/').pop()].filter(key => !!key);
```

Comment:
Should we use `path.sep` similar to [getPackageName](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/reporters/spec-reporter/wdio-spec-reporter.js#L128) instead of `/`?

Reply:
done here https://github.com/cerner/terra-toolkit/pull/617/commits/6b8f2b8d59fce9ff47c70329be39d2968fb4c3cf#diff-4b2b0085c896f72621885e9dfd3550a5ea69fc1516f63f9bc4a21a607cc97b34R37